### PR TITLE
CHOP-12. Add equals and hashcode methods to KijiTap.

### PR DESCRIPTION
**Pull request by robotoer**
@lockoff, @daisyzhou.

### CODE REVIEW

File: src/main/java/com/wibidata/lang/KijiScheme.java [hunk 1]
```diff
@@ -176,6 +177,21 @@ public void sinkPrepare(FlowProcess<JobConf> flowProcess,
...
+  @Override
+  public boolean equals(Object other) {
+    if (!(other instanceof KijiScheme)) {
+      return false;
+    }
 
-  // TODO: Implement equals, hashCode.
+    final KijiScheme scheme = (KijiScheme) other;
+    return Objects.equal(mRequest, scheme.mRequest)
+        && Objects.equal(mColumns, scheme.mColumns);
```
Comment (daisyzhou) [inline src/main/java/com/wibidata/lang/KijiScheme.java:189]: mColumns doesn't exist in KijiScheme yet, did you mix this up with a different change you had to make first?

Comment (daisyzhou) [inline src/main/java/com/wibidata/lang/KijiScheme.java:189]: I will change this to mOutputSpec and push for now. (It's a Map<String, String> so using .equal should be fine)

Comment (robotoer) [inline src/main/java/com/wibidata/lang/KijiScheme.java:189]: ah, yeah, oops.

**Comment by daisyzhou** [comment]
pushed af082b69b6ce6ba488109e29f429ab41c0fe2409

**Pull request closed without merging**